systemtest/src/main/java/io/strimzi/systemtest/cli/KafkaCmdClient.java

```diff
@@ -4,84 +4,112 @@
  */
 package io.strimzi.systemtest.cli;
 
-import io.strimzi.api.kafka.model.KafkaResources;
 import io.strimzi.systemtest.utils.kafkaUtils.KafkaTopicUtils;
 import org.apache.logging.log4j.Level;
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
-import static io.strimzi.systemtest.resources.ResourceManager.kubeClient;
 import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
 
 public class KafkaCmdClient {
 
-    private static final int PORT = 9092;
-
-
-    public static List<String> listTopicsUsingPodCli(String namespaceName, String clusterName, int kafkaPodId) {
-        String podName = KafkaResources.kafkaPodName(clusterName, kafkaPodId);
-        return Arrays.asList(cmdKubeClient(namespaceName).execInPod(podName, "/bin/bash", "-c",
-            "bin/kafka-topics.sh --list --bootstrap-server localhost:" + PORT).out().split("\\s+"));
-    }
-
-    public static List<String> listTopicsUsingPodCli(String clusterName, int kafkaPodId) {
-        return listTopicsUsingPodCli(kubeClient().getNamespace(), clusterName, kafkaPodId);
+    public static List<String> listTopicsUsingPodCli(String namespaceName, String scraperPodName, String bootstrapServer) {
```

Comment:
Do we want to use `scraperPodName` or something more generic?

Reply:
More generic will be better. Thanks :)